internal/webhooks/cluster_test.go

```diff
@@ -992,7 +992,21 @@ func TestClusterTopologyValidationWithClient(t *testing.T) {
 				Build(),
 			class: builder.ClusterClass(metav1.NamespaceDefault, "clusterclass").
 				Build(),
-			wantErr: true,
+			wantErr: false,
+		},
+		{
+			name: "Accept a cluster with non-existent clusterclass referenced cluster.spec.topology.class",
```

Comment:
Just out of curiosity: Is the idea to test this once with "wrongName" and once with "#"?

Reply:
This is an unnecessary duplicate for sure.